packages/vite/src/vite.ts

```diff
@@ -63,6 +63,9 @@ export async function bundle (nuxt: Nuxt) {
           copyPublicDir: false,
           rollupOptions: {
             output: {
+              sourcemapIgnoreList: (relativeSourcePath) => {
+                return relativeSourcePath.includes('/node_modules/') || relativeSourcePath.includes(ctx.nuxt.options.buildDir)
```

Comment:
Does this work on Windows? For `rollup` I had to change it to `relativeSourcePath.includes('node_modules')` to make the test pass on the Windows bot.

Reply:
updated in https://github.com/nuxt/nuxt/commit/73ade185b2e5d5fa1b2d659cb5c8f7d8d63a8ac5